src/allocs_profile.jl

```diff
@@ -143,8 +139,8 @@ function to_pprof(alloc_profile::Profile.Allocs.AllocResults
         # for each location in the sample.stack, if it's the first time seeing it,
         # we also enter that location into the locations table
         location_ids = UInt64[
-            maybe_add_location(location)
-            for location in sample.stacktrace
+            maybe_add_location(frame)
+            for frame in sample.stacktrace if (!frame.from_c || from_c) && (frame.func != :Array)
```

Comment:
```suggestion
            for frame in sample.stacktrace if (!frame.from_c || from_c)
```
I'd probably leave off this :Array piece for now, and let people do it in the UI.

We can think about whether there's a more general way to do this though... 🤔 If we can't think of one.. i think maybe we can do what you suggest, even though it's lame.